.github/workflows/cmake.yml

```diff
@@ -88,6 +107,14 @@ jobs:
 #    - name: Clear ccache
 #      run: ccache --clear
 
+    - name: create sopt package on gcc
+      if: ${{ contains(matrix.cxx, 'g++-9') }}
+      run: conan create ${{github.workspace}}/sopt --build missing -s:b compiler.libcxx=libstdc++11 -o:b mpi=${{matrix.mpi}} -o:b openmp=${{matrix.omp}}
+
+    - name: create sopt package on apple-clang
+      if: ${{ contains(matrix.cxx, 'clang++') }}
+      run: conan create ${{github.workspace}}/sopt --build missing -o:b mpi=${{matrix.mpi}} -o:b openmp=${{matrix.omp}}
+
```

Comment:
Should we build SOPT with examples and/or test off just to save some compile time?

Reply:
Examples off sounds like a good idea to me, not sure about tests... maybe you would still like to be able to test sopt separately.